### PR TITLE
Fix typo in id_ID Address

### PR DESCRIPTION
**Pull request by pattisahusiwa**
This PR:

- [ ] Adds a new feature ...
- [ ] Covered by tests
- [x] Fixes #nnnn

### CODE REVIEW

File: src/Faker/Provider/id_ID/Address.php [hunk 1]
```diff
@@ -85,15 +85,15 @@ class Address extends \Faker\Provider\Address
 
     /**
      * Abbreviated State Names.
-     * Currently this is all just a guesswork, but should be highly accurate
+     *
```
Comment (krsriq) [inline src/Faker/Provider/id_ID/Address.php:88]: How about adding a reference (@link) here?

Comment (krsriq) [reply src/Faker/Provider/id_ID/Address.php:88]: Thanks! 
I just cross-referenced the list with the one from Wikipedia - there's Kaltara missing in in stateAbbr and I can't find Kalut in Wikipedia. 
That's probably two names for the same province? Still, the list should definitely be identical to the one in the provided link, so could you please either find a link where Kaltara is replaced with Kalut, or fix it in the list?

Comment (pattisahusiwa) [reply src/Faker/Provider/id_ID/Address.php:88]: Done. It's hard to find a single page that contains the acronyms of all the states in Indonesian language. The acronyms on wiki page are accurate and, IMHO, can be used as a reference.

Comment (pattisahusiwa) [reply src/Faker/Provider/id_ID/Address.php:88]: Yes, you're right. The abbreviation of Kalimantan Utara is Kaltara.

Comment (krsriq) [reply src/Faker/Provider/id_ID/Address.php:88]: Thanks!

**Review by bramceulemans — APPROVED** [review]

**Comment by bramceulemans** [comment]
Looks good to me!

**Pull request merged**